backend/de.metas.business/src/main/java/de/metas/order/impl/OrderLineBL.java

```diff
@@ -773,6 +773,12 @@ public ProductPrice getCostPrice(final org.compiere.model.I_C_OrderLine orderLin
 				.build();
 	}
 
+	@Override
+	public BigDecimal getPriceActual(final org.compiere.model.I_C_OrderLine orderline)
```

Comment:
as said earlier: returning a BigDecimal as a price without any context is dangerous. instead, return `ProductPrice`

Reply:
removed that completely, using getcostprice() which returns a Productprice with price actual and uomID